### PR TITLE
add docs for UI

### PR DESCRIPTION
**Pull request by lsmith77**
what else do you think we should add at this point @solutionDrive-Alt ?

### CODE REVIEW

**Comment by solutionDrive-Alt** [comment]
Maybe some sort of Definition how the files being imported should be structured. But on the other hand is this maybe not the right location for that, because it could be a lot of information when we get more importers.

**Comment by solutionDrive-Alt** [comment]
I think we should definitely add information about the current supported file-types. And the already implemented Importers should imho also be listed. So there is no misunderstanding why importer 'xy' is present but importer 'z' is not

**Comment by solutionDrive-Alt** [comment]
Where should we put informations for other developers, like what steps have to be done, to implement a new importer?

**Comment by lsmith77** [comment]
added some more docs based on your feedback

File: README.md [hunk 1]
```diff
@@ -36,6 +36,20 @@
...
 ## Usage
 
+### Available importer types
+
+* payment_method (csv)
+* tax_category (csv)
+
+## Example import files
+
+See the fixtures in the Behat tests: `tests/Behat/Resources/fixtures`
```
Comment (solutionDrive-Alt) [inline README.md]: nice idea! 👍

**Comment by lsmith77** [comment]
if its ok for you like this @solutionDrive-Alt, then lets merge :)

**Pull request merged**